### PR TITLE
Optimize:optimize BranchStatus annotation

### PR DESCRIPTION
**Pull request by lightClouds917**
Ⅰ. Describe what this PR did
Optimize BranchStatus.java annotation,the annotation is very messy.

### CODE REVIEW

**Comment by lightClouds917** [comment]
the second commit id to add log for  issue #1400

File: server/src/main/java/io/seata/server/coordinator/DefaultCore.java [hunk 1]
```diff
@@ -82,6 +82,7 @@ public Long branchRegister(BranchType branchType, String resourceId, String clie
                 globalSession.addBranch(branchSession);
             } catch (RuntimeException ex) {
                 branchSession.unlock();
+                LOGGER.error("Failed to add branchSession to globalSession : " + ex);
```
Comment (slievrly) [inline server/src/main/java/io/seata/server/coordinator/DefaultCore.java]: +ex ？

Comment (lightClouds917) [reply server/src/main/java/io/seata/server/coordinator/DefaultCore.java]: Just add error log,  easy to find problems,The real cause is still being investigated

Comment (xingfudeshi) [inline server/src/main/java/io/seata/server/coordinator/DefaultCore.java]: Please use LOGGER.error("Failed to add branchSession to globalSession:{}",ex.getMessage(),ex);

Comment (lightClouds917) [reply server/src/main/java/io/seata/server/coordinator/DefaultCore.java]: Ok,Thank for your patience,I have modified it,but,I read a lot of error logs in seata,The style is not same.for example:
LOGGER.error("transaction[" + tx.getXid() + "] retry fetch status times exceed the limit [" + RETRY_MAX_TIMES + " times]");
LOGGER.error("fetch GlobalTransaction status error", e);
LOGGER.error("error occurred while getting value{}", e.getMessage());
LOGGER.error("not support metrics exporter type: " + exporterTypeName, exx);
......
they all used at catch{LOGGER.erro},unify the writing style will be better.

Comment (xingfudeshi) [reply server/src/main/java/io/seata/server/coordinator/DefaultCore.java]: Yes.The problem does exist,will optmize in the future.

**Pull request merged**